### PR TITLE
[Documentation] Fixed logo customization 

### PR DESCRIPTION
**Pull request by maciek-wozniak**
| Q               | A
| --------------- | -----
| Branch?         | 1.9 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | 
| Related tickets |
| License         | MIT

Fixed path to logo template file

### CODE REVIEW

File: docs/getting-started-with-sylius/shop-customizations.rst [hunk 1]
```diff
@@ -24,8 +24,8 @@ Default logo in shop panel:
 The first step is to detect which template is responsible for displaying the logo and therefore which should be overridden
 to customize a logo image.
 
-It's placed in **SyliusShopBundle**, at ``Resources/views/_header.html.twig.path``, so to override it,
-you should create the ``templates/bundles/SyliusShopBundle/_header.html.twig`` file and copy the original file content.
+It's placed in **SyliusShopBundle**, at ``Resources/views/Layout/Header/_logo.html.twig.path``, so to override it,
```
Comment (Zales0123) [inline docs/getting-started-with-sylius/shop-customizations.rst:27]:
```suggestion
It's placed in **SyliusShopBundle**, at ``Resources/views/Layout/Header/_logo.html.twig`` path, so to override it,
```

**Review by Zales0123 — COMMENTED** [review]
In fact, we could even mention, it can be customized by changing the configuration of `sylius.shop.layout.header.grid` UI event, but still, this change definitely fixed the bug :)

**Comment by Zales0123** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "docs/fixing-logo-customization"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by Zales0123 — APPROVED** [review]

**Comment by CoderMaggie** [comment]
Thanks, @maciek-wozniak! :tada:

**Pull request merged**